packages/perseus/src/widgets/numeric-input/numeric-input.test.ts

```diff
@@ -467,45 +478,162 @@ describe("transform", () => {
-        const transform = NumericInputWidgetExport.transform;
-        const widgetOptions: PerseusNumericInputWidgetOptions = {
+describe("interactive: full vs answerless", () => {
+    beforeAll(() => {
+        registerAllWidgetsForTesting();
+    });
+
+    let userEvent: UserEvent;
+    beforeEach(() => {
+        userEvent = userEventLib.setup({
+            advanceTimers: jest.advanceTimersByTime,
+        });
+
+        jest.spyOn(Dependencies, "getDependencies").mockReturnValue(
+            testDependencies,
+        );
+    });
+
+    it("is answerable with full widget options", async () => {
+        // Arrange
+        const fullNumericInputOptions: PerseusNumericInputWidgetOptions = {
             coefficient: false,
             static: false,
             size: "normal",
             answers: [
                 {
                     status: "correct",
                     maxError: null,
-                    strict: true,
-                    value: 0.5,
-                    simplify: "required",
-                    answerForms: ["proper"],
-                    message: "",
-                },
-                {
-                    status: "wrong",
-                    maxError: null,
-                    strict: true,
-                    value: 0.5,
+                    strict: false,
+                    value: 42,
                     simplify: "required",
-                    answerForms: ["decimal"],
                     message: "",
+                    answerForms: ["proper", "improper", "mixed"],
                 },
             ],
         };
-        const renderProps = transform(widgetOptions);
-        expect(renderProps).toEqual({
+
+        const fullItem: PerseusRenderer = {
+            content: "[[☃ numeric-input 1]] ",
+            images: {},
+            widgets: {
+                "numeric-input 1": {
+                    type: "numeric-input",
+                    options: fullNumericInputOptions,
+                },
+            },
+        };
+
+        // Act
+        const {renderer} = renderQuestion(fullItem);
+        await userEvent.tab();
+        expect(screen.getByRole("textbox")).toHaveFocus();
+
+        // Assert
+        expect(
+            screen.getByText(/a simplified proper fraction, like 3\/5, or/),
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(/a simplified improper fraction, like 7\/4, or/),
+        ).toBeInTheDocument();
+        expect(
+            screen.getByText(/a mixed number, like 1 and 3\/4/),
+        ).toBeInTheDocument();
+
+        await userEvent.type(screen.getByRole("textbox", {hidden: true}), "42");
+
+        const userInput = renderer.getUserInputMap();
+        const score = scorePerseusItem(fullItem, userInput, "en");
+
+        // Assert
+        expect(score).toHaveBeenAnsweredCorrectly();
+    });
+
+    it("is interactive with answerless widget options", async () => {
```

Comment:
Here's the important test. We:

- Render the widget with the stripped data
- Assert that UI that previously required `answers` still renders
- Answer and score the widget